operator/pkg/test/controller/customization.go

```diff
@@ -224,6 +224,46 @@ var NamespacedControllerResourceCRWrongNamespace = &customizev1alpha1.Namespaced
 	},
 }
 
+var (
+	ValidatingWebhookCRForDuplicatedWebhook = &customizev1alpha1.ValidatingWebhookConfigurationCustomization{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "validating-webhook",
+		},
+		Spec: customizev1alpha1.WebhookConfigurationCustomizationSpec{
+			Webhooks: []customizev1alpha1.WebhookCustomizationSpec{
+				{
+					Name: "deny-immutable-field-updates", // a valid webhook name
+				},
+				{
+					Name: "resource-validation", // another valid webhook name
+				},
+				{
+					Name: "deny-immutable-field-updates", // a valid webhook name but duplicated
+				},
+			},
+		},
+	}
+	MutatingWebhookCRForDuplicatedWebhook = &customizev1alpha1.MutatingWebhookConfigurationCustomization{
```

Comment:
Is MutatingWebhookCRForDuplicatedWebhook used somewhere? I cannot find it.

Reply:
Good catch! I forgot to add a unit test case.